internal/mgrutils/reports.go

```diff
@@ -28,10 +28,7 @@ func RunReport(ctx context.Context, kubeCfg *rest.Config, kongCfg sendconfig.Kon
 	}
 
 	// create a universal unique identifier for this system
-	uuid, err := uuid.GenerateUUID()
-	if err != nil {
-		return fmt.Errorf("failed to generate a random uuid: %w", err)
-	}
+	uuid := uuid.NewString()
```

Comment:
As required by the unchecked checkbox in the PR description: please add a changelog entry to CHANGELOG.md under the "TBD" release that we're changing the uuid generation mechanism.